src/components/dropdown/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { ComponentProps } from 'react'
+import classNames from 'classnames'
+import styles from './styles.module.css'
+
+export type SelectedOption = string | readonly string[] | number | undefined
+
+interface Props extends ComponentProps<'select'> {
+  id: string
+  className?: string
+  options: ComponentProps<'option'>[]
+  label: string
+  hiddenLabel?: boolean
+  selectedOption?: SelectedOption
+  onChangeValue: (selectedOption: SelectedOption) => void
+}
+
+const Dropdown = ({ id, className, label, hiddenLabel, options, selectedOption, onChangeValue, ...rest }: Props) => {
+  const handleOptionChange = (e: React.ChangeEvent<HTMLSelectElement>): void => {
+    const newOption = e.target.value as SelectedOption
+    onChangeValue(newOption)
+  }
+
+  return (
+    <>
+      <label hidden={hiddenLabel} htmlFor={`dropdown-${id}`}>
```

Comment:
les infos hidden ne sont pas lu par les lecteurs d'ecran 
https://www.w3.org/WAI/tutorials/forms/labels/#hiding-the-label-element